src/main/java/org/sagebionetworks/bridge/exporter/handler/HealthDataExportHandler.java

```diff
@@ -217,6 +228,20 @@ protected void setTsvInfoForTask(ExportTask task, TsvInfo tsvInfo) {
         return rowValueMap;
     }
 
+    /**
+     * post process tsv to call update records' exporter status as SUCCEEDED
+     * @param tsvInfo
+     */
+    @Override
+    protected void postProcessTsv(TsvInfo tsvInfo) throws BridgeExporterException {
+        List<String> recordIds = tsvInfo.getRecordIds();
+        if (recordIds == null) {
+            throw new BridgeExporterException("No record id in this tsv file.");
```

Comment:
well yes i think the null is not permitted here. I changed the initialization to create a new arraylist instead of a null